daprdocs/content/en/getting-started/configure-state-pubsub.md

```diff
@@ -0,0 +1,232 @@
+---
+type: docs
+title: "How-To: Configure state store and pub/sub message broker"
+linkTitle: "Configure state & pub/sub"
+weight: 40
+description: "Configure state store and pub/sub message broker components for Dapr"
+aliases:
+  - /getting-started/configure-redis/
+---
+
+In order to get up and running with the state and pub/sub building blocks two components are needed:
+
+1. A state store component for persistence and restoration
+2. As pub/sub message broker component for async-style message delivery
+
+A full list of supported components can be found here:
+- [Supported state stores]({{< ref supported-state-stores >}})
+- [Supported pub/sub message brokers]({{< ref supported-pubsub >}})
+
+The rest of this page will describe how to get up and running with Redis.
+
+{{% alert title="Self-hosted mode" color="warning" %}}
+When initialized in self-hosted mode, Dapr will automatically run a Redis container and setup the required component yaml files. You can skip this page and go to [quickstarts]({{< ref quickstarts.md >}})
```

Comment:
Since the next step is either quickstarts or setting up dev environment maybe the link can just point to the next steps section below

Reply:
Can we change some of the "will" to present tense?  For example this one "The rest of this page will describe how to get up and running with Redis." to "The rest of this page describes how to get up and running with Redis."

Reply:
Generally can we reduce the number of "will"s to present tense usage instead.

Reply:
Pushed and contributing guide updated